openapi-spec/api.yaml

```
@@ -0,0 +1,811 @@
+        lastName:
+          type: string
+          example: "Frusciante"
+        avatarUrl:
+          type: string
+          format: uri
+          description: "S3 url to the user's avatar image."
+        roles:
+          type: array
+          items:
+            type: string
+          example: [ "STUDENT" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)